prov/rxm/src/rxm_init.c

```diff
@@ -166,6 +166,23 @@ static int rxm_init_info(void)
 	return 0;
 }
 
+static void rxm_alter_info(const struct fi_info *hints, struct fi_info *info)
+{
+	struct fi_info *cur;
+
+	if (!hints)
+		return;
```

Comment:
Should these be removed if no hints were provided, since they do affect the fast path?